src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -310,6 +310,66 @@ public void castIntFieldToDoubleWithAliasJdbcFormatGroupByTest() {
         }
     }
 
+    @Test
+    public void functionAbsInWhereClauseWithIsNullTest() {
+        JSONObject response = executeJdbcRequest(
+                "SELECT age FROM " + TEST_INDEX_ACCOUNT + " WHERE (ABS(balance) IS NULL)"
```

Comment:
So all your IT use `TEST_INDEX_ACCOUNT ` index for testing, I wonder is there any document in this index missing `balance` or `firstname` field? Because I recall the painless scripts we generate are unable to handle missing field (https://github.com/opendistro-for-elasticsearch/sql/issues/308), I assume your IT actually doesn't cover the typical use of `IS NULL` or `IS NOT NULL`.

Reply:
Good point- I'll look for/create an index with some missing field values

Reply:
One suggestion, instead of create new index, could we consider using the existing index. I think there are already 10+ different indices in place.